### PR TITLE
changelog and helm updates for 0.6.0

### PR DESCRIPTION
**Pull request by tesshuflower**
**Describe what this PR does**
- updates the CHANGELOG.md with updates for the 0.6.0 release.
- also updates the helm chart to version 0.6.0 and updates gpg signing key
**Is there anything that requires special attention?**

**Related issues:**

### CODE REVIEW

**Comment by tesshuflower** [comment]
/cc @JohnStrunk 

Do you mind taking a look John?

File: CHANGELOG.md [hunk 1]
```diff
@@ -183,7 +197,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support for rsync & rclone replication
 - Helm chart to deploy operator
 
-[Unreleased]: https://github.com/backube/volsync/compare/v0.5.0...HEAD
+[Unreleased]: https://github.com/backube/volsync/compare/release-0.6...HEAD
+[0.6.0]: https://github.com/backube/volsync/compare/release-0.5...v0.6.0
```
Comment (JohnStrunk) [inline CHANGELOG.md:201]: We should probably tag and release a 0.5.1 at some point

**Review by JohnStrunk — COMMENTED** [review]
Add yourself to the maintainers list in `Chart.yaml`

**Review by JohnStrunk — COMMENTED** [review]
/lgtm

**Pull request merged**